userdocs/src/usage/customizing-the-kubelet.md

```diff
@@ -48,12 +48,21 @@ nodeGroups:
 
 In this example, given instances of type `m5a.xlarge` which have 4 vCPUs and 16GiB of memory, the `Allocatable` amount
 of CPUs would be 3.4 and 15.4 GiB of memory. In addition, the `DynamicKubeletConfig` feature gate is also enabled. It is
-important to know that the values specified in the config file for the the fields in `kubeletExtraconfig` will 
+important to know that the values specified in the config file for the the fields in `kubeletExtraconfig` will
 completely overwrite the default values specified by eksctl. However, omitting one or more `kubeReserved` parameters
 will cause the missing parameters to be defaulted to sane values based on the aws instance type being used.
 
-!!!warning
-    By default `eksctl` sets `featureGates.RotateKubeletServerCertificate=true`, but when custom `featureGates` are provided, it will be unset. You should always include 
-    `featureGates.RotateKubeletServerCertificate=true`, unless you have to disable it.
- 
+### A note on the `kubeReserved` calculation for NodeGroups with mixed instances
+
+While it is generally recommended to configure a mixed instance NodeGroup to use instances with the same CPU and RAM
+configuration; that's not a strict requirement. Therefore the `kubeReserved` calculation uses the _smallest instance_ in
+the `InstanceDistribution.InstanceTypes` field. This way NodeGroups with disparate instance types will not reserve too
+many resources on the smallest instance. However, this could lead to a reservation that is too small for the largest
+instance type.
+
+### Warning
+By default `eksctl` sets `featureGates.RotateKubeletServerCertificate=true`, but when custom `featureGates` are
+provided, it will be unset. You should always include `featureGates.RotateKubeletServerCertificate=true`, unless
+you have to disable it.
```

Comment:
```suggestion
!!! Warning
    By default `eksctl` sets `featureGates.RotateKubeletServerCertificate=true`, but when custom `featureGates` are
provided, it will be unset. You should always include `featureGates.RotateKubeletServerCertificate=true`, unless
you have to disable it.